apps/web/app/(auth)/auth/verification-requested/page.tsx

```diff
@@ -3,31 +3,37 @@
 import { RequestVerificationEmail } from "@/components/auth/RequestVerificationEmail";
 import { useSearchParams } from "next/navigation";
 import FormWrapper from "@/components/auth/FormWrapper";
+import validator from "validator";
```

Comment:
validator has a gzipped bundle size of 33,8kb that also needs to be sent to the client because it's a client component 😥
https://bundlephobia.com/package/validator@13.11.0

I think we should find a better solution for that. Maybe just use a regex validation for the email or move this page to a server component and use an existing library like `zod` serverside to keep the bundle size low?
The page should work as a server component, as we can also get the searchParams as props:
https://nextjs.org/docs/app/api-reference/file-conventions/page

Reply:
okay, on it!

Reply:
Hey @mattinannt, I have made the changes ie:
- Moved the page to server side component
- Used the zod lib for input validation
- Removed the introduction of the validator lib

Have tested it locally as well with a few happy and sad cases. Please take a look